portfolio/src/main/webapp/script.js

```diff
@@ -102,20 +102,119 @@ function getComments() {
 class Countries {
 
   constructor(jsonObj) {
-    this.keyedJSON = jsonObj;
+
+    //  = JSON.parse(JSON.stringify(jsonObj));
+    this.keyedJSONBackup = JSON.parse(JSON.stringify(jsonObj));
```

Comment:
Any reason you need to still maintain both `this.keyedJSON` and `this.keyedJSONBackup`?

Reply:
The ***keyedJSONBackup*** contains Antarctica countries/locations. ***keyedJSON*** loses the locations when ever it merges with and continent JSON object. So if the user ever reselects all continents it will contain Antarctica data.